index.html

```diff
@@ -91,7 +91,8 @@
             <div class="ui-block-b"><button type="submit" data-theme="a" id="b-submit">Submit</button></div>
           </fieldset>
 	  <br>
-	  <p>OpenHAB for Pebble by Richard Lee</p>
+	  <p>openHAB Pebble application by Richard Lee</p>
```

Comment:
the more contributions are coming from other community members, you might want to consider to change it to "by Richard Lee and others" or "by the openHAB community".